data/converters/converters.go

```diff
@@ -0,0 +1,195 @@
+package converters
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+)
+
+func toConversionError(expected string, v interface{}) error {
+	return fmt.Errorf("expected %s input but got type %T for value \"%v\"", expected, v, v)
```

Comment:
```suggestion
	return fmt.Errorf(`expected %s input but got type %T for value "%v"`, expected, v, v)
```

Back ticks can be used in Go for quoting to avoid the escaping of quotes.